begoneads/begoneadsGUI.py

```diff
@@ -0,0 +1,322 @@
+# Copyright  (c) 2022 Cid0rz (cid.kampeador@gmail.com)
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
```

Comment:
This is not needed, the license is not included in the source files in this repository.